src/common/schedulers/slurm_commands.py

```diff
@@ -362,15 +367,21 @@ def _parse_nodes_info(slurm_node_info):
         "State": "state",
         "Partitions": "partitions",
         "Reason": "reason",
+        "SlurmdStartTime": "slurmdstarttime",
     }
 
-    node_info = slurm_node_info.split("######")
+    node_info = slurm_node_info.split("######\n")
     slurm_nodes = []
     for node in node_info:
-        lines = node.strip().splitlines()
+        lines = node.splitlines()
         kwargs = {}
         for line in lines:
             key, value = line.split("=")
+            if key == "SlurmdStartTime":
+                if value != "None":
```

Comment:
minor: line 381- 384 can be simplify into one line: `value = datetime.strptime(value, "%Y-%m-%dT%H:%M:%S").astimezone(tz=timezone.utc) if value != "None" else None`

Reply:
Unfortunately the line becomes too long and the autoformatter splits it into even more lines that it's using now. I'm going to keep it as it is.